### DISCUSSION TITLE
Is this email from GitHub spam

### DISCUSSION THREAD

**Discussion by 1Mark** [community/community#43417] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I got this email supposedly from GitHub but it wasn't the email associated to my GitHub account but possibly a 3 years ago it was....

![Screenshot_20230105-193445](https://user-images.githubusercontent.com/15360245/210865841-a58e78b3-dc4c-4bb5-bf8e-348499805bb3.png)

It was from docs-research@github.com

Is this email legit?

**Comment by queenofcorgis** [comment, marked as answer]
Hi there @1Mark 👋🏼 , 

Thanks for reaching out, I appreciate you being so vigilant. I can confirm this is a legitimate GitHub email from our content designers. 

Cheers,
Lili
